### PR TITLE
[KOGITO-2489] - Migrate kogito-cloud-operator PR check to RHBA jenkins

### PR DESCRIPTION
**Pull request by Kaitou786**
See: https://issues.redhat.com/browse/KOGITO-2489
Signed-off-by: Tarun Khandelwal <tarkhand@redhat.com>

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,22 +1,31 @@
...
+            jdk 'kie-jdk11'
+        }
     environment {
+        TEMP_TAG="""pr-${sh(
+                returnStdout: true,
+                script: 'echo \${GIT_COMMIT} | cut -c1-7'
+            ).trim()}"""
+        JAVA_HOME = "${GRAALVM_HOME}"
+
         WORKING_DIR = "/home/jenkins/go/src/github.com/kiegroup/kogito-cloud-operator/"
```
Comment (radtriste) [inline Jenkinsfile:23]: I don't think we need to be in a specific `WORKING_DIR` anymore

Comment (Kaitou786) [reply Jenkinsfile:23]: okay, I'm not sure about this. I'll remove this and see if it works =D

**Comment by Kaitou786** [comment]
@ricardozanini Currently no we don't have the way to export the logs publicly, but Enrique is working on that. Besides our current CI on AWS is not reliable, it fails without any reasons, this migration will give more stability to our CI

**Comment by radtriste** [comment]
@Ginxo any update on @Kaitou786 comment ?

**Comment by spolti** [comment]
I would agree with @ricardozanini , not sure if it is a good idea to migrate it at this point without have the option to externalize the CI logs from the rhba.

**Comment by Ginxo** [comment]
@radtriste nothing for the moment. We performed several PoCs but nothing is decided for the moment. We can provide you more details at the end of the week

**Comment by ricardozanini** [comment]
Clearing the milestone since we won't do this migration until the logs are publicly available.

**Comment by Kaitou786** [comment]
Closing as currently, we are not choosing this as a solution, [this](https://issues.redhat.com/browse/KOGITO-3064) can be a solution

**Pull request closed without merging**